File: app/src/main/java/com/artmurka/artmurkaapp/Model/InterfacesModel/IBasket.java
package com.artmurka.artmurkaapp.Model.InterfacesModel;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemBasket.BasketItems;

import io.reactivex.Observable;
import retrofit2.Call;


public interface IBasket {
    Observable<BasketItems> toBasket(String goodId);
    Observable<BasketItems> getItemInBasket();
    Call<BasketItems> deleteItemFromBasket(String goodId);

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/interfacesmodel/IBasket.kt
package com.artmurka.artmurkaapp.data.model.interfacesmodel

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket.BasketItems

import io.reactivex.Observable
import retrofit2.Call


interface IBasket {
    val itemInBasket: Observable<BasketItems>
    fun toBasket(goodId: String): Observable<BasketItems>
    fun deleteItemFromBasket(goodId: String): Observable<BasketItems>
}
